pkgs/by-name/du/dumpifs/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  clang,
+  fetchFromGitHub,
+  lib,
+  lzo,
+  lz4,
+  stdenv,
+  ucl,
+  unstableGitUpdater,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dumpifs";
+  version = "0-unstable-2020-05-07";
+
+  src = fetchFromGitHub {
+    owner = "askac";
+    repo = "dumpifs";
+    rev = "b7bac90e8312eca2796f2003a52791899eb8dcd9";
+    hash = "sha256-vFiMKcPfowLQQZXlXbq5ZR1X6zr7u3iQwz3o4A6aQMY=";
+  };
+
+  buildInputs = [
+    clang
+    lzo
+    lz4
+    ucl
+    zlib
+  ];
+
+  postUnpack = ''
+    rm source/{dumpifs,exMifsLzo,uuu,zzz}
+  '';
+
+  patches = [ ./package.patch ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 dumpifs exMifsLz4 exMifsLzo fixdecifs fixencifs uuu zzz -t $out/bin
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "Tool for those who are interested in hacking MIB2 firmware";
+    homepage = "https://github.com/askac/dumpifs";
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [ KSJ2000 ];
```

Comment:
You also need to add yourself to maintainer-list.nix :)

Reply:
I added myself in unyaffs pull request. After it is merged, the others should be good as well.

Reply:
Added in https://github.com/NixOS/nixpkgs/pull/348979, waiting on [a rebase](https://github.com/NixOS/nixpkgs/pull/348979#issuecomment-2475643855)

Reply:
Done for:
https://github.com/NixOS/nixpkgs/pull/348766
https://github.com/NixOS/nixpkgs/pull/349032
https://github.com/NixOS/nixpkgs/pull/349055
https://github.com/NixOS/nixpkgs/pull/348765
https://github.com/NixOS/nixpkgs/pull/348768